test/std/process/exec_test.ds

```diff
@@ -1,6 +1,11 @@
+if is_windows
+    command = set "cmd /c"
+else
+    command = set "sh -c"
```

Comment:
lets drop the else. the command will be undefined and not part of the exec i believe.
that way for linux/mac it will execute:
exec echo hello world 
without the sh -c prefix.

Reply:
I was entangled in the need to modify  the operating mode of  other environments before, as you said, it's not very meaningful. I ended up adding these because it looks symmetrical

Reply:
So now I am thinking about whether there is the meaning of this pr, because the test cases here just need to prove that the  part of the Rust code (the function) is no problem. When that part of the code is correct, all platforms will produce the same result (because Rust is cross-platform )

Reply:
the problem we are trying to solve is the one you raised, and that exec echo is failing on windows 10.
so if i'm using it in my tests, for windows i need to prefix with cmd as you suggested. but no real need to do it for linux/mac.

Reply:
I just rethinked this question again. The code itself has no problems. The result is only affected by the environment. Whether duckscript Command `exec` (Rust code `Command::new()`) can find the relevant executable files in the environment.
I installed the mingw64 environment in my windows system and generated most of the executable files, such as echo.exe, true.exe, and then run `cargo make`, passed the test directly.
So I think there is no need to make any changes because of the execution environment. (In fact, `cargo fmt` currently does not work well on windows because of the windows path.)